### PR TITLE
Issue 2893 - CLI - dscreate - add options for setting up replication

### PR DESCRIPTION
**Pull request by mreynolds389**
Description:

Add options for setting up replication to dscreate

relates: https://github.com/389ds/389-ds-base/issues/2893

### CODE REVIEW

File: src/lib389/lib389/instance/options.py [hunk 1]
```diff
@@ -357,4 +358,39 @@ def __init__(self, log, section):
...
+        self._type['replica_bindgroup'] = str
+        self._helptext['replica_bindgroup'] = "Set the replication bind group DN"
+
+        self._options['changelog_max_age'] = "7d"
+        self._type['changelog_max_age'] = str
+        self._helptext['changelog_max_age'] = ("How long an entry should remain in the replication changelog (requires that replication is enabled).")
+
+        self._options['changelog_max_entries'] = "-1"
+        self._type['changelog_max_entries'] = str
+        self._helptext['changelog_max_entries'] = ("The maximum number of entries to keep in the replication changelog (requires that replication is enabled).")
```
Comment (Firstyear) [inline src/lib389/lib389/instance/options.py]: -1 is no limit, defaults to no limit

**Review by Firstyear — DISMISSED** [review]

**Comment by Firstyear** [comment]
Minor change in text, otherwise looks good :)

File: src/lib389/lib389/instance/options.py (hunk 1 above)
Comment (mreynolds389) [reply src/lib389/lib389/instance/options.py]: Fixed

**Review by droideck — COMMENTED** [review]
The code looks good to me!

One thing only, could you please add a simple test while you're on it?
Something like https://github.com/389ds/389-ds-base/blob/master/dirsrvtests/tests/suites/basic/basic_test.py#L1472

**Comment by mreynolds389** [comment]
> The code looks good to me!
> 
> One thing only, could you please add a simple test while you're on it? Something like https://github.com/389ds/389-ds-base/blob/master/dirsrvtests/tests/suites/basic/basic_test.py#L1472

@droideck Done!

**Review by droideck — APPROVED** [review]
Awesome! LGTM!

**Pull request merged**